controllers/remote/cluster.go

```diff
@@ -21,19 +21,19 @@ import (
 
 	"github.com/pkg/errors"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 	restclient "k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
 	kcfg "sigs.k8s.io/cluster-api/util/kubeconfig"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 )
 
 // ClusterClientGetter returns a new remote client.
-type ClusterClientGetter func(ctx context.Context, c client.Client, cluster *clusterv1.Cluster, scheme *runtime.Scheme) (client.Client, error)
+type ClusterClientGetter func(ctx context.Context, c client.Client, cluster types.NamespacedName, scheme *runtime.Scheme) (client.Client, error)
```

Comment:
Should this also be `client.ObjectKey` here?

Reply:
Yeah, absolutely, needs a fix :+1: